4788/src/main/include/ControlMap.h

```diff
@@ -80,9 +83,8 @@ struct ControlMap {
   inline static const wml::controllers::tAxis drivebaseR{Driver, XboxController::kRightYAxis};
   
   // Climber
-   static constexpr int ClimberPort = 0;
-   static constexpr int ClimberSolenoidPort1 = 0;
-   static constexpr int ClimberSolenoidPort2 = 0;
+ inline static const wml::controllers::tButton ClimberToggle{ CoDriver, XboxController::kA };
```

Comment:
indentation not standard to the rest of file. (1 space less)

Reply:
Fixed